pylint/utils.py

```diff
@@ -81,7 +81,7 @@
 # Allow stopping after the first semicolon/hash encountered,
 # so that an option can be continued with the reasons
 # why it is active or disabled.
-OPTION_RGX = re.compile(r"\s*#.*\bpylint:\s*([^;#]+)[;#]{0,1}")
+OPTION_RGX = re.compile(r'.*#\s*pylint:\s*([^;#]+)[;#]?.*')
```

Comment:
I think replacing `.*` with `\s*` from after the hash makes sense, but not the rest of the changes.